WebInstall/explainer_cross_domain.md

```diff
@@ -150,9 +150,13 @@ To install a same domain web site/app, the process is as follows:
  
 ### The `navigator.getInstalledApps` method
 
-If supported by the UA, the `getInstalledApps` method returns a list of the content that has been installed from that installation origin.** This means the installation origin will be able to know which applications it has installed, until cache is cleared. The installation origin *will not* be informed of any apps installed by other means, whether via another installation origin, directly through the browser, or by a native app store. The method returns a list of manifest ids of content installed from the calling origin. Additionally, if the browser has an active 'Do Not Track (DNT)', equivalent 'Global Privacy Control (GPC)' setting, is in Private browsing mode, or is an opinionated browser towards privacy, this is ignored and installation origins will not be allowed to know if that application is installed. 
+If supported by the UA, the `getInstalledApps` method returns a list of the content that has been installed from *that* installation origin. This is an **async** method of the `navigator` interface that allows the installation origin to know which applications it has installed.
```

Comment:
Is this returning the list of _all_ apps that this installation origin every installed (since the last cache purge), or just the apps installed from this installation origin that _are still installed_?  If `foo.com` installs `bar.com` and then the user uninstalls `bar.com`, when `foo.com` calls `navigator.getInstalledApps()`, will it be told about `bar.com`?

Reply:
IMO, if we should either:
- only return the _currently installed_ apps that were installed by the installation origin, OR
- remove this API if the additional fingerprinting surface is a concern.

My concern is that stores/catalogs will use the value returned here to hide an install button, which could be confusing for users if they uninstalled an app, but it is listed in this store as still being installed so they're not given the option to install it again.

Reply:
@amandabaker by *currently installed* you mean @HowardWolosky 's comment to apps that are *still installed?

If the UA knows which apps are installed based on `manifest_id` then I would expect the call to return the apps that match and are still installed indeed. 

It should only return currently installed apps *at time of (method) calling*. This makes it harder to fingerprint since this list is dynamic and can change between sessions/visits and also it's flexible enough that a store could update the UX of a site base on actual information of currently installed apps at the time of calling.

Reply:
>@amandabaker by currently installed you mean @HowardWolosky 's comment to apps that are still installed? 

yep! And what you described SGTM, especially since this introduces noise that makes it harder to fingerprint a user. In that case, wdyt about:
```suggestion
If supported by the UA, the `getInstalledApps` method returns a list of the content that has been installed from *that* installation origin which is still installed on the device at the time of execution. This is an **async** method of the `navigator` interface that allows the installation origin to know which applications it has installed.
```

Reply:
I've committed your suggestion. It reflects what we are thinking for the method. Sent a re-review. :)